_posts/2015-01-12-transatlantic.md

```diff
@@ -9,78 +9,76 @@ credit: Charlie Marshall
 
-The UK has been fighting net migration, trying to reduce immigration. **Being part of the EU, the UK cannot limit EU nationals from entering.** Thus: the easiest group to limit admittance from is non-EU nationals. (Or the UK could leave the EU, but let’s leave that discussion aside for now.)
+**The fight against immigration is one that wins votes, but rarely makes economic sense.**  The UK has been fighting net migration, trying to reduce immigration. **Being part of the EU, the UK cannot limit EU nationals from entering.** The easiest group to keep out is non-EU nationals.
 
 ### Restricting British families
 
 In 2012 UK Parliament updated immigration rules for British citizens with non-EU spouses. British citizens [must make at least £18,600/year to sponsor their partners](http://www.theguardian.com/law/2014/jul/11/appeal-court-18600-foreign-spouse-uk). This was--and still is--a controversial rule. On minimum wage, a British citizen would not earn enough to sponsor their spouse. Earning a decent wage in a poorer country excludes UK expats from returning with their families.
 
 <span id="r-1"></span>
-I am not a fan of this rule. Fortunately, my girlfriend makes enough to qualify us. We're great immigrants<sup>[1](#footnote-1)</sup> as we're both creating new jobs in the UK.
+I am not a fan of this rule. Fortunately, my girlfriend makes enough to qualify us. (What a great moneybags girl! You should marry the pants off her!) Together, we're a great immigration candidate<sup>[1](#footnote-1)</sup> as we're both creating new jobs in the UK.
 
 ## Proof of income
 
 If your sponsor (British citizen) lives and works in the UK, proving your income is easy. Even if your sponsor is self-employed in the UK, the process isn't hard.
 
-But: if you're a British citizen working outside the UK, you'll need more documentation. A **lot** more.
+But if you're a British citizen working outside the UK, you'll need more documentation. A **lot** more.
 
 **It is possible to apply when both partners are outside the UK.** That wasn't even clear to me when I started. Here's how we did it:
 
 ### Self-Employment Evidence
 
-First thing's first: the self-employed sponsor should be able to "move" their business. _Without the intent to move their business, UK Immigration will refuse your application._ Proving this is as simple as writing it in a letter, but it's worth mentioning.
+First things first: the self-employed sponsor should be able to "move" their business. _Without the intent to move the business, UK Immigration will refuse your application._ Proving this is as simple as writing it in a letter, but it's worth mentioning.
```

Comment:
See, I will go with your suggestion, but "First thing is first" totally makes sense so I don't feel like my version is wrong. It's entirely possible it's not the turn of phrase, but that could just be me misappropriating the meaning in my head.